### PR TITLE
AUTO-191: Update email address for clinician and user

### PR DESCRIPTION
**Pull request by ginnyyadav**
Updated email address for clinician, custodial and data user storage accounts. Will change the variables in travis as soon as this is merged. (note: these tests will be failing on travis right now since the variables have not been update there and lenny is currently debugging tidewhisperer on qa). Runs fine locally.

### CODE REVIEW

File: tests/API_contract_tests.postman_collection.json [hunk 1]
```diff
@@ -1656,20 +1654,20 @@
...
 							"script": {
-								"id": "d7808f80-2e6d-4863-bbed-58bc063a59f2",
+								"id": "b906c075-b943-4d3f-b8cf-a583ca993ed6",
 								"exec": [
-									"const moment = require('moment');",
-									"pm.variables.set(\"currentTimestamp\", moment().unix());",
 									"const uuid = require('uuid');",
-									"pm.variables.set(\"signupPw\", uuid.v4().slice(0,9));"
+									"pm.variables.set(\"signupPw\", uuid.v4().slice(0,9));",
+									"let dataUserEmailAddress = \"automated-tests+user\"+ Date.now() + \"@tidepool.org\";",
```
Comment (jh-bate) [inline tests/API_contract_tests.postman_collection.json]: I think these, email addresses, should come from travis config rather than checked into code

Comment (jh-bate) [reply tests/API_contract_tests.postman_collection.json]: see https://github.com/tidepool-org/APITests#cli section where we pass in emails from config

**Review by jh-bate — CHANGES_REQUESTED** [review]

File: tests/API_contract_tests.postman_collection.json (hunk 1 above)
Comment (jh-bate) [reply tests/API_contract_tests.postman_collection.json]: **login email declared in environment:**
https://github.com/tidepool-org/APITests/blob/master/tests/QA1.postman_environment.json#L11

**login email variable used in tests:**
https://github.com/tidepool-org/APITests/blob/master/tests/API_contract_tests.postman_collection.json#L35

**and then travis config declares variable that is pulled from config**
https://travis-ci.com/github/tidepool-org/APITests/builds/190372489/config

**was:**
`\"automated-tests+user\"+ Date.now() + \"@tidepool.org\";"`

**could be in config:**
`automated-tests+user_datetag_@tidepool.org`

**then converts in code:**
`let uniqeEmail = {{fromConfig}}.replace('_datetag_', Date.now())`

Comment (ginnyyadav) [reply tests/API_contract_tests.postman_collection.json]: thanks jamie. great suggestion. Also, did that for the sharing email address we have in the smoke test

**Review by jh-bate — APPROVED** [review]
LGTM

**Pull request merged**